### DISCUSSION TITLE
How to sync a clock to the daw

### DISCUSSION THREAD

**Discussion by dinegredin** [DISTRHO/Cardinal#474] [Q&A]
Hello,
I have difficulties to sync to my daw ( Ableton or Bitwig ).
I’m trying to sync Impromptu Clocked module to the Host Time module but it won’t work.
It doesn’t seem to have a BBT option on the clock module.
Is there a technique I’m missing ?
Thanks for your time

**Comment by dromer** [comment]
Use the Host Time `Step` output and set the mode of Impromptu Clocked to `P4`

**Reply by dinegredin** [reply]
Thanks ! It works indeed !

**Comment by falkTX** [comment]
Yesh the "Host Time" module is your starting point. the beat, bar and steps trigger based on host time.

**Reply by dinegredin** [reply]
👌
